airbyte-server/src/test/java/io/airbyte/server/helpers/ConnectionHelpers.java

```diff
@@ -40,6 +39,12 @@
   private static final String BASIC_SCHEDULE_TIME_UNIT = "days";
   private static final long BASIC_SCHEDULE_UNITS = 1L;
 
+  public static final io.airbyte.config.ResourceRequirements TESTING_RESOURCE_REQUIREMENTS = new io.airbyte.config.ResourceRequirements()
+      .withCpuLimit("100g")
```

Comment:
These values are for unit tests, so I was setting them intentionally super high so they wouldn't be accidentally used elsewhere. Any integration tests that want to control these shouldn't be setting them from this test helper class. Added a comment.

Reply:
I understand that they you wan them high but is 100g a valid value for a cpu limit? should it just be 100? or it this saying you want 100K cpus? feel free to just re-resolve if i'm wrong, but wanted to make sure the unit was intentional.

Reply:
I believe it uses the same structure as Kubernetes memory assignments, even though in practice you would only use partial values like `100m`. 

Really it doesn't matter what is here. The values could be `klajsdfklafsdklfjsadklj` because these should only be used to check that the values are propagated correctly by unit tests and not actually used to launch something on Kubernetes.